modules/ROOT/pages/values-and-types/temporal.adoc

```diff
@@ -1090,11 +1095,9 @@ RETURN d.weeks, d.days, d.daysOfWeek
 ======
 
 
-.+duration+
+.Extract the most significant second based components of a `DURATION` value
```

Comment:
I don't understand what this sentence means, neither how the example compares to the one after.

Reply:
I don't fully understand what this does, but from an editorial point of view, it should be _second-based_.

Reply:
ye. having some trouble here as well
d.minutes is 61, so probably it adds the 60 minutes from the hour that was set in the query. likewise for the other return values.
the day is not calculated as 24 hours, because it's not second-based (this again only makes sense after looking at the table in "components of durations" - there are three component groups (months, days, seconds, the latter of which contains hours, minutes, seconds, and so forth).

there's another table after that one, explaining the less significant components of these groups.

it's... not easy to get as is.

Reply:
added some explanations to several queries, named the relevant tables and added references to them